yt_dlp/extractor/twitch.py

```diff
@@ -383,10 +437,41 @@ def _extract_info_gql(self, info, item_id):
             'was_live': True,
         }
 
+    def _extract_storyboard(self, item_id, storyboard_json_url, duration):
+        spec = list(self._download_json(storyboard_json_url, item_id, "Downloading storyboard metadata JSON") or [])
+        # sort from highest quality to lowest
+        spec.sort(key=lambda x: int_or_none(x.get('width')) or 0, reverse=True)
```

Comment:
No need to sort. Core code should handle it

Reply:
Then sb1 will(might) be lower quality than sb0. I'd rather it be consistent with youtube (sb0 - highest quality, sb1 - lower, etc)

Reply:
Makes sense. Could u explain that in the comment coz the intention is not obvious